src/core/src/main/java/org/apache/jmeter/gui/action/LookAndFeelCommand.java

```diff
@@ -95,10 +95,16 @@ private static MenuItem ofDarklafTheme(Theme theme) {
     static {
         if (System.getProperty("darklaf.decorations") == null) {
             System.setProperty("darklaf.decorations", "false");
+        } else if ("true".equals(System.getProperty("darklaf.allowNativeCode"))) {
```

Comment:
Have you had a look at `Boolean.getValue(name)`? It should do the same as the `"true".equals(System.getProperty(...)`.